code/modules/client/preferences.dm

```diff
@@ -1338,6 +1369,22 @@ var/list/preferences_datums = list()
 			switch(href_list["preference"])
 
 				//citadel code
+				if("genital_colour")
+					switch(features["genitals_use_skintone"])
+						if(TRUE)
+							features["genitals_use_skintone"] = FALSE
+						if(FALSE)
+							features["genitals_use_skintone"] = TRUE
+							features["cock_color"] = sanitize_hexcolor(skintone2hex(skin_tone))
+							features["breasts_color"] = sanitize_hexcolor(skintone2hex(skin_tone))
+							features["vag_color"] = sanitize_hexcolor(skintone2hex(skin_tone))
+							features["balls_color"] = sanitize_hexcolor(skintone2hex(skin_tone))
```

Comment:
so you mean the actual colour in their features is the same as before, but it should **display** a skintone to the user instead? I think I can do that. Should end up making the skintone able to toggle off and on that way also, since right now it only toggles TO skintone and kind of sticks there when you toggle off and need manual change....